### PR TITLE
Improve notes on CMake error with container

### PR DESCRIPTION
**Pull request by alberk8**
## Description
When building the devcontainer you might encounter error message at Step 8.   The error message is "Bad Cmake executable".

## Motivation and Context

Fix cannot go pass Step 8 of the instruction.

## Types of changes

- [x] Improvement (correction, content improvement, typoe fix, formating)
- [ ] New Article (new document for docs website)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:

- [x] My doc follows the code style of this project.
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.

### CODE REVIEW

File: content/building/using-dev-container.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Go to the file `.devcontainer/devcontainer.json` and choose the container you wa
 
 > Note: as a beginner, we recommend you to use the pre built containers. If you are familiar with containers and need to adjust resources in them, then use the one which you can build yourself. Use the one in the `./sources` subdirectory. So the file name will be for example `./source/Dockerfile.All` to use the container containing everything and build it from the source.
 
+> Error: if you get error message like "Bad Cmake executable "". Is it installed or settings contain the correct path (cmake.cmakePath)?  The solution: uninstall the Cmake and Cmake tool from Visual Studio Code and restart Visual Studio Code.
```
Comment (Ellerbach) [inline content/building/using-dev-container.md:40]:
```suggestion
> Error: if you get error message like `Bad Cmake executable. Is it installed or settings contain the correct path (cmake.cmakePath)?`  The solution: uninstall Cmake and Cmake tool from Visual Studio Code and restart Visual Studio Code.  
```

**Review by Ellerbach — APPROVED** [review]
Just a little proposition for the text. If ok for you, just commit it and I'll merge. Thanks!

**Comment by Ellerbach** [comment]
@alberk8 I have been running few more test to validate your issue. And I've adjusted the text.
In short, the issue is because the build folder is first filled by your local Windows Cmake extensions. Once you open in the container, if you don't clean it, you'll get errors.
Also, please sign the licence so we can merge with the new text.

**Comment by josesimoes** [comment]
@alberk8 I'm committing @Ellerbach suggestion above and merge this. OK?

**Comment by josesimoes** [comment]
@alberk8 please sign the CLA...

**Pull request merged**